shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -92,5 +128,13 @@ object Time {
   }
 
   /** Current time */
-  def now(): Time = ???
+      def now() : Time = {
```

Comment:
Actually I was just giving it a try to change to System.currentTimeMillis() but found this on JavaDocs, which means we might end up having bad values if we don't consider checking on various OS's then, 

_"Note that while the unit of time of the return value is a millisecond, the granularity of the value depends on the underlying operating system and may be larger. For example, many operating systems measure time in units of tens of milliseconds."_

What do you think?